addremove.test.js

```diff
@@ -0,0 +1,113 @@
+import {
+  buildTask, addTask, addNewTask, deleteTask,
+} from './src/modules/addremove'; // Import the functions to be tested
+// import todoTasks from './src/modules/taskDB';
```

Comment:
- [OPTIONAL] I'll suggest you remove this as leaving unused code as comment t in your codebase is against the best practices and also makes your codebase look a bit unprofessional. :+1: :+1: